crates/packet_inspector_gui/README.md

```diff
@@ -0,0 +1,77 @@
+# What's This?
```

Comment:
We should just point people to the README in `packet_inspector` instead of maintaining 2 documents. Feel free to update that document to mention `packet_inspector_gui`.

Reply:
I think we should just replace the old packet inspector entirely with this project.

Reply:
In my experience, the old packet inspector introduces a few bugs - for example, I've had it not send chat messages until the player moves and not send block interact events until the player hits the ground :/.
Given that, I'm not sure that keeping the old one around is even worth it. Perhaps adding a feature to disable the GUI and log the packets instead in this new project would be a better idea?

Reply:
> I think we should just replace the old packet inspector entirely with this project.

For me, CLI tools much more useful than GUI applications, so I would not appreciate the removal of `packet_inspector` package.

Reply:
Then plan was to incorporate the CLI tool with this, with a `--nogui` or something cli switch